src/extra/normalize.src.css

```diff
@@ -138,7 +138,7 @@
   border-radius: var(--radius-2);
 }
 
-:where(textarea) { 
+:where(textarea) {
```

Comment:
this is a strange commit? can you remove it so the PR is just on the index.html file?

Reply:
huh! That must have been from prettier. I must have pressed `ctrl + shift + i` to format the file manually. My bad.

Reply:
Looking at it now... I don't know how to actually update it without creating a commit in that file.

![image](https://github.com/user-attachments/assets/a1768a98-734c-4a0d-8406-62df7b70a33d)

Although I pulled from my remote now and I can see that index.html has untracked changes. Perhaps pushing that change may update `normalize.src.css`? 

![image](https://github.com/user-attachments/assets/68229531-863e-4cf7-99bc-514e040a180b)


What do you think?

Reply:
i'll squash this work in, so the history isn't that big of a deal. if you just push an update to the branch that removes this commit, that should be good enough. i'm not a stickler about the file history, and yeah, we'll be squashing so shouldnt be an issue.

Reply:
oh okay. i'll dod this now. 

got stuck on a couple of things, hence the reason I'm replying now 🙏🏼